client/verta/verta/data_types/_line.py

```diff
@@ -0,0 +1,33 @@
+import numbers
+
+from .._internal_utils import arg_handler
+
+from . import _VertaDataType
+
+
+class Line(_VertaDataType):
+    _TYPE_NAME = "line"
+    _VERSION = "v1"
+
+    @arg_handler.args_to_builtin(ignore_self=True)
+    def __init__(self, x, y):
+        if len(x) != len(y):
+            raise ValueError("`x` and `y` must have the same length")
+        if not all(isinstance(el, numbers.Real) for el in x):
```

Comment:
maybe we want a `util.contains_only_reals(x)` function? Seems we're repeating that check in a few places

Reply:
7b7d3d1